### ISSUE TITLE
test(a11y): insufficient contrast on colored notifications

### ISSUE THREAD

**Issue by mfranzke** [db-ui/core#702]
The contrast ratio in between text- and background-color on some notifications is slightly or much too low, like e.g. _informative_, _success_ and _warning_:

![image](https://github.com/db-ui/core/assets/787658/f1d7db37-d0b9-49ef-850b-6d4d7990708a)

[WCAG 1.4.3 Contrast (Minimum)](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html)
